layout/explore/explore-detail/component.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Components
+import Spinner from 'components/ui/Spinner';
+import ExploreDetailHeader from './explore-detail-header';
+
+// Styles
+import './styles.scss';
+
+class ExploreDetailComponent extends React.Component {
+  static propTypes = {
+    dataset: PropTypes.object.isRequired,
```

Comment:
Getting this error when entering the details component 

`Warning: Failed prop type: The prop dataset is marked as required in ExploreDetailComponent, but its value is null`

if it can be null it should probably not have the `isRequired` defined or make sure it's not sent as `null`

Reply:
Fixed here: https://github.com/resource-watch/resource-watch/pull/1253/commits/140357cf63e93432504a902b5ef7eea870a4a441